src/style-spec/reference/v8.json

```diff
@@ -3681,8 +3681,8 @@
     "range": {
       "type": "array",
       "default": [
-        5,
-        10
+        2,
+        12
```

Comment:
These seem like reasonable defaults to me. For reference, here's [2,12]:

![fog-2-12 copy](https://user-images.githubusercontent.com/572717/114078472-d5b93f80-985d-11eb-8639-af8060820382.jpg)

and [5,12]:
![fog-5-12](https://user-images.githubusercontent.com/572717/114078482-d94cc680-985d-11eb-9c86-06654ac5a82f.jpg)